src/Equinox.MessageDb/MessageDb.fs

```diff
@@ -372,10 +372,16 @@ type AccessStrategy<'event, 'state> =
     /// It should return the event case whose name matches <c>snapshotEventCaseName</c>
     /// </summary>
     | AdjacentSnapshots of snapshotEventCaseName: string * toSnapshot: ('state -> 'event)
+    /// <summary>
+    /// Calls the provided <c>project</c> function to update a table in the same transaction as the appended events
```

Comment:
FYI `<c>` is not actually handled correctly by the compiler xmldoc gen (so VS, VSCode but also not Rider dont render it differntly; IIRC it does nothing)
(learned via Abel in TaskSeq - it's quite a tire fire)
(`paramref` is, which is why the xmldoc of that abuses it so much)

Reply:
Renders fine for me? Or at least it's rendered as monospace
<img width="919" alt="Screenshot 2024-01-09 at 09 31 47" src="https://github.com/jet/equinox/assets/3721521/06696705-69f8-4244-a137-9826266a24ca">

Reply:
That seems correct (had not actually personally checked as I always believed it worked) - unfortunately I don't use VS/VSCode often...
So let's stick with the `<c>`s...